model_security_adjust_sst/security/stock_security.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="group_stock_operator" model="res.groups">
+        <field name="name">Operator</field>
+        <field name="category_id" ref="base.module_category_warehouse_management"/>
+        <field name="implied_ids" eval="[(4, ref('stock.group_stock_user'))]"/>
+    </record>
+
+    <record id="stock.group_stock_manager" model="res.groups">
+        <field name="implied_ids"
+               eval="[(3, ref('stock.group_stock_user')), (4, ref('group_stock_operator'))]"/>
+    </record>
+
+    <record id="operator_stock_move_perm" model="ir.rule">
+        <field name="name">operator: stock.move permission</field>
+        <field name="model_id" ref="stock.model_stock_move"/>
+        <field name="groups" eval="[(4, ref('group_stock_operator'))]"/>
+        <field name="perm_read" eval="True"/>
+        <field name="perm_write" eval="True"/>
+        <field name="perm_create" eval="True"/>
+        <field name="perm_unlink" eval="True"/>
+    </record>
```

Comment:
This record is not needed. The record in ir.model.access.csv does the job.

Reply:
This part is deleted